kie-wb-tests/kie-wb-tests-gui/src/test/java/org/kie/wb/selenium/ui/StandalonePerspectivesIntegrationTest.java

```diff
@@ -0,0 +1,121 @@
+
+            put("AdministrationPerspective", By.className("qe-list-bar-header-Repository-Editor"));
+            put("AdminPagePerspective", By.id("admin-page-title"));
+            put("AppsPerspective", By.cssSelector("[title='Create Directory']"));
+            put("Asset Management", By.className("qe-list-bar-header-Repository-Configuration"));
+            put("AuthoringPerspective", By.className("fa-refresh"));
+            put("AuthoringPerspectiveNoContext", By.cssSelector("[title='Project Explorer']"));
+            put("ContentManagerPerspective", By.className("qe-list-bar-header-Content-Manager"));
+            put("DashboardPerspective", ByJQuery.selector("span:contains('Processes & Tasks Dashboard')"));
+            put("DataSetAuthoringPerspective", By.className("qe-list-bar-header-Data-Set-Authoring-Home"));
+            put("DataSourceManagementPerspective", By.className("qe-list-bar-header-DataSource-Explorer"));
+            put("ExecutionErrors", By.className("qe-list-bar-header-Execution-Errors"));
+            put("GuvnorM2RepoPerspective", By.className("qe-list-bar-header-M2-Repository-Content"));
+            put("HomePerspective", ByJQuery.selector("h2:contains('The Knowledge Life Cycle')"));
+            put("LibraryPerspective", By.id("welcome"));
+            put("PlugInAuthoringPerspective", ByJQuery.selector("h3:contains('Plugin Explorer')"));
+            put("PreferencesCentralPerspective", By.className("preferences-editor"));
+            put("ProcessDefinitions", By.className("qe-list-bar-header-Process-Definitions"));
+            put("ProcessInstances", By.className("qe-list-bar-header-Process-Instances"));
+            put("Requests", By.className("qe-list-bar-header-Requests-List"));
+            put("SecurityManagementPerspective", By.className("qe-list-bar-header-Security-management"));
+            put("ServerManagementPerspective", By.className("fa-refresh"));
+            put("SocialHomePagePerspective", By.className("qe-list-bar-header-Recent-Assets"));
+            put("TaskAdmin", By.className("qe-list-bar-header-Task-Administration-List"));
+            put("Tasks", By.className("qe-list-bar-header-Tasks-List"));
+            put("UserHomePagePerspective", By.className("fa-home"));
+            put("WiresGridsDemoPerspective", By.className("qe-list-bar-header-Grids"));
+            put("WiresTreesPerspective", By.className("qe-list-bar-header-Palette"));
+        }
+    };
+
+    @Test
+    public void loadStandalonePerspectives() {
+        login.getLoginPage();
+        login.loginDefaultUser();
+
+        for (Map.Entry<String, By> entry : perspectiveToElement.entrySet()) {
+            String perspectiveId = entry.getKey();
+            By elementInPerspective = entry.getValue();
+
+            loadPerspectiveAndCheckLoadingTime(perspectiveId, true);
+            verifyPerspectiveIsLoaded(perspectiveId, elementInPerspective);
+            verifyPresenceOfHeader(true);
```

Comment:
From the point of test design, `headerIncluded` is definitely two different test cases, one for `true` and one for `false`.